### PR TITLE
Issue #6532: aligned javadoc/xdoc for NeedBraces

### PR DESCRIPTION
**Pull request by pbludov**
Issue #6532 (part of #5750)

No changes in the code.

### CODE REVIEW

**Comment by rnveach** [comment]
@pbludov test is failing.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java [hunk 1]
```diff
@@ -29,107 +29,98 @@
...
- *     default: counter = 100; break; // OK
- * }
+ *   case 1: counter++; break; // OK
+ *   case 6: counter += 10; break; // OK
+ *   default: counter = 100; break; // OK
  * }
  * </pre>
  * <p>
- * To configure the Check to allow {@code while, for} loops with empty bodies:
+ * To configure the check to allow loops ({@code while, for}) with empty bodies:
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java]: Other places you have captial `Check`, so why was this one made lowercase? Can we have 1 style throughout the document?

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java [hunk 2]
```diff
@@ -29,107 +29,98 @@
...
- * {@code
- * for (int i = 0; ; ) this.notify();
- * }
- * </p>
- * <p><b>allowEmptyLoopBody</b> which allows loops with empty bodies, e.g.:</p>
- * <p>
- * {@code
- * while (value.incrementValue() < 5);
- * }
+ * To configure the check to allow single-line statements
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java]: Same.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java [hunk 3]
```diff
@@ -29,107 +29,98 @@
...
- * An example of how to configure the check is:
+ * To configure the check:
  * </p>
  * <pre>
  * &lt;module name="NeedBraces"/&gt;
  * </pre>
- * <p> An example of how to configure the check for {@code if} and
- * {@code else} blocks is:
+ * <p>
+ * To configure the check for {@code if} and {@code else} blocks:
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java]: Same.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java [hunk 4]
```diff
@@ -29,107 +29,98 @@
...
+ * LITERAL_FOR</a>,
+ * <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#LITERAL_IF">
+ * LITERAL_IF</a>,
+ * <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#LITERAL_WHILE">
+ * LITERAL_WHILE</a>.
+ * </li>
+ * </ul>
  * <p>
- * An example of how to configure the check is:
+ * To configure the check:
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java]: Same.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java (hunk 1 above)
Comment (pbludov) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java]:
```
$ grep -rl "configure the Check"
src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/CommentsIndentationCheck.java
src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java
src/main/java/com/puppycrawl/tools/checkstyle/checks/design/VisibilityModifierCheck.java
src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java
src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheck.java
```
Only in five files the `Check` is in uppercase. I'm going to fix all of them.

**Comment by pbludov** [comment]
@rnveach Done. Switched to lower case for this check and `EmptyCatchBlock`.

**Pull request merged**